express/storage/base_storage.js

```diff
@@ -26,14 +27,17 @@ class BaseStorage {
     }
 
     async hget(key, hashKey) {
+        logger.warn("This method will be deprecated in future.");
```

Comment:
Please update the [README](https://github.com/gofynd/fdk-extension-javascript/blob/main/express/storage/README.md) of the depreciation as well.